.travis.yml

```
@@ -20,12 +20,12 @@ install:
         echo "Full Installation";
         pip install -r develop_requirements.txt;
       fi
-    - python setup.py develop
+    - pip install -e .
 
 script:
     - ls
     - pytest --version
-    - coverage run --source="./qtt" --omit="*/test_*"  -m pytest
+    - coverage run --source="./src" --omit="*/test_*"  -m pytest
```

I guess we can remove the omit part now?
```suggestion
    - coverage run --source="./src"  -m pytest
```

---

Already done.